### DISCUSSION TITLE
[Question] Installing django-filter with Poetry

### DISCUSSION THREAD

**Discussion by davidvalaz** [django-json-api/django-rest-framework-json-api#972] [Q&A]
I see in the readme we're told to install **django-filter** with the following command:
```bash
pip install djangorestframework-jsonapi['django-filter']
```

I am using Poetry and this command doesn't work. Can I just use the following instead?
```bash
poetry add django-filter
```

**Comment by sliverc** [comment]
This seems to have been fixed in poetry version 1.1.3 with https://github.com/python-poetry/poetry/issues/3129 so poetry supports `extra_requires` as well.

But of course if it does not work you can also install the django-filter dependency manually as well:

```
poetry add djangorestframework-jsonapi
poetry add django-filter
```
